### PR TITLE
Automated tests for indexes and materialized views [ORACLE]

### PR DESCRIPTION
**Pull request by chetank-yb**

### CODE REVIEW

File: migtests/lib/yb.py [hunk 1]
```diff
@@ -129,3 +129,8 @@ def fetch_datatypes_of_all_tables_in_schema(self, schema_name="public") -> Dict[
 				tables[table_name] = []
 			tables[table_name].append(data_type)
 		return tables
+
+	def refresh_materialized_view(self, view_name, schema_name="public") -> None:
+		cur = self.conn.cursor()
+		cur.execute(f"REFRESH MATERIALIZED VIEW {schema_name}.{view_name}")
+		self.conn.commit()
```
[the remark this answers is not in the document]
Comment (priyanshi-yb) [reply migtests/lib/yb.py:145]: Is there any need to have this `refresh_materialized_view()` function seperate, as I mentioned we can run this particular query using `run_query_and_chk_error(query, None)`

[the remark this answers is not in the document]
Comment (chetank-yb) [reply migtests/lib/yb.py:146]: @priyanshi-yb , this has been fixed now

[the remark this answers is not in the document]
Comment (priyanshi-yb) [reply migtests/lib/yb.py]: Okay, Thanks!

**Review by priyanshi-yb — APPROVED** [review]
LGTM

**Pull request merged**